src/gui/locale/message_en.json

```diff
@@ -35,6 +35,8 @@
   },
   "replay": {
     "done-title": "Replay is done",
+    "done-run-operations": "Replay is finished.",
+    "done-run-operations-because-skipped": "Replay was canceled because it was paused during recording.",
```

Comment:
`Replay was interrupted because the test contains some unrecorded operations due to capture pause.`でお願いします。

また、上記に合わせて、日本語も`「記録の一時停止」により記録されなかった操作があるため、履歴再生を中止しました。`としてください。

Reply:
指摘通り、修正致しました。